### DISCUSSION TITLE
VOR needles in ARC mode

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4010] [General]
Hello simmers,

I´am using the latest custom FBW mod. Are the VOR needles in ARC mode implemented or not? I can´t see them. Bug or on the to do list?

Thanx and greets 

Klaus

**Comment by tracernz** [comment]
Not merged yet https://github.com/flybywiresim/a32nx/pull/3886. I have a couple of fix-ups to do, then they should proceed to QA testing.
